sheetstorm/management/commands/constants.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+from typing import Dict
+
+from users.models import CustomUser
+
+SUPERUSER_USER_TYPE = "SUPERUSER"
+
+
+class TestDataSize(Enum):
+    SMALL = "small"
+    MEDIUM = "medium"
+    LARGE = "large"
+    EXTRA_LARGE = "extra_large"
+
+
+class ProjectType(Enum):
+    SUSPENDED = "suspended"
+    ACTIVE = "active"
+    COMPLETED = "completed"
+
+
+quantities_in_small_set = 10
+quantities_in_medium_set = 70
+quantities_in_large_set = 300
+quantities_in_extra_large_set = 1000
+
+
+def create_set_of_data_size(data_quantity: int) -> Dict[str, int]:
+    return {
+        SUPERUSER_USER_TYPE: 1,
+        CustomUser.UserType.ADMIN.name: data_quantity,
+        CustomUser.UserType.EMPLOYEE.name: data_quantity,
+        CustomUser.UserType.MANAGER.name: data_quantity,
+        ProjectType.SUSPENDED.name: data_quantity,
+        ProjectType.ACTIVE.name: data_quantity,
+        ProjectType.COMPLETED.name: data_quantity,
+    }
```

Comment:
That is not what we want to create. We don't want to have in every data set the same amount of projects and users. There should be 4 different datasets where for example in small dataset we will have 5 opened, 1 suspended and 2 completed projects - about users the same situation - 2 admins, 3 managers and 15 employees - something like this :)